data/json/recipes/other/materials.json

```diff
@@ -32,8 +32,7 @@
       [ [ "forge", 80 ], [ "char_forge", 80 ] ]
     ],
     "components": [
```

Comment:
Attempts at cleanup are very welcome. If you proceed one file at a time then the risk of merge conflict is acceptably low.

For recipes you can sort via:
`./tools/cleanup.sh recipe data/json/foo.json`

For items you can sort via:
`./tools/cleanup.sh item data/json/foo.json`

Keep linting as a separate commit to any other (preferably minimal) changes you make